### PR TITLE
Update SiriusXM presence and add Deno

### PR DESCRIPTION
**Pull request by SnoweyFoxx**
image
image
image
image

### CODE REVIEW

**Comment by SnoweyFoxx** [comment]
Resolves #1787 
image
image
image
image
image

File: websites/D/Deno.land/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "id": "604182077088202784"
+  },
+  "service": "Deno.land",
+  "description": {
+    "en": "Deno is a runtime for JavaScript and TypeScript that is based on the V8 JavaScript engine and the Rust programming language. It was created by Ryan Dahl, original creator of Node.js, and is focused on productivity."
+  },
+  "url": ["deno.land", "doc.deno.land"],
+  "version": "1.0.1",
+  "logo": "https://deno.land/logo.svg",
+  "thumbnail": "https://i.imgur.com/a/LInQbj5.png",
```
Comment (doomlerd) [inline websites/D/Deno.land/dist/metadata.json:13]: The link is wrong.

File: websites/D/Deno.land/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "author": {
+    "name": "Alexx",
+    "id": "604182077088202784"
+  },
+  "service": "Deno.land",
+  "description": {
+    "en": "Deno is a runtime for JavaScript and TypeScript that is based on the V8 JavaScript engine and the Rust programming language. It was created by Ryan Dahl, original creator of Node.js, and is focused on productivity."
+  },
+  "url": ["deno.land", "doc.deno.land"],
+  "version": "1.0.1",
```
Comment (doomlerd) [inline websites/D/Deno.land/dist/metadata.json:11]: Should be 1.0.0

File: websites/D/Deno.land/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "author": {
+    "name": "Alexx",
+    "id": "604182077088202784"
+  },
+  "service": "Deno.land",
```
Comment (doomlerd) [inline websites/D/Deno.land/dist/metadata.json:6]: Name is "Deno" only

**Comment by doomlerd** [comment]
now both links are wrong lmao

**Pull request merged**